src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -332,6 +332,82 @@ public async Task TestMailDeliveryAsync()
             }
         }
 
+        [Fact]
+        public void TestSmtpDeliveryFormatSevenBit() {
+            SmtpServer server = null;
+            SmtpClient client = null;
+            var serverList = new System.Collections.Generic.List<SmtpServer>();
+
+            MailMessage msg = new MailMessage("foo@example.com", "bar@example.com");
+            msg.HeadersEncoding = msg.BodyEncoding = msg.SubjectEncoding = System.Text.Encoding.UTF8;
+
+            var subjectTxt = "Test 测试 Contain 包含 UTF8";
+            var subjectEncode = "=?utf-8?B?VGVzdCDmtYvor5UgQ29udGFpbiDljIXlkKsgVVRGOA==?=";
+
+            msg.Subject = subjectTxt;
+
+            Action<SmtpDeliveryFormat, bool, string, string, string> test = (format, utf8, async, sync, tag) => {
+                var retry = 0;
+                while (retry < 3) {
+                    if (client != null) {
```

Comment:
Please change bracing style to match the CoreFx standard (i.e. bracing style is Allman-style where braces are on their own lines).